src/main/java/teammates/common/datatransfer/questions/FeedbackRankRecipientsQuestionDetails.java

```diff
@@ -238,6 +239,79 @@ public String getQuestionAdditionalInfoHtml(int questionNumber,
                         Slots.QUESTION_ADDITIONAL_INFO, additionalInfo);
     }
 
+    private List<FeedbackResponseAttributes> getActualResponses(
+            FeedbackQuestionAttributes question,
+            FeedbackSessionResultsBundle bundle) {
+        List<FeedbackResponseAttributes> responses;
+        String questionId = question.getId();
+        //Get all actual responses for this question.
+        responses = new ArrayList<>();
+        for (FeedbackResponseAttributes response : bundle.actualResponses) {
+            if (response.feedbackQuestionId.equals(questionId)) {
+                responses.add(response);
+            }
+        }
+        return responses;
+    }
+
+    private String getStudentQuestionResultsStatisticsHtml(
```

Comment:
Add a Javadoc although it is a private method.
State the behaviour of the method.